workers/unity/Packages/com.improbable.gdk.mobile/iOS/Utility/DeviceInfo.cs

```diff
@@ -0,0 +1,12 @@
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile.Ios
+{
+    public class DeviceInfo
+    {
+        public static bool IsIosSimulator()
```

Comment:
Wondering if it may be better to have a common abstract base class that can be extended per different platforms? It's not really clear how the `Is*Emulator/Simulator` API will be used though, so this could probably be refactored in beta phase.

Reply:
Agree. It is hard to say what would be preferable for developers, but the current structure of having core Mobile assembly + platform specific iOS and Android assemblies should allow us to refactor classes the way we need when we understand the requirements better